src/redux/wishlist/wishlistActions.js

```diff
@@ -0,0 +1,37 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from '../../services/axios/';
+
+export const getWishlistItems = createAsyncThunk(
```

Comment:
do we still need getWishlistItems when we have getSortedWishlistItems ?

Reply:
I don't think so)) will remove it

Reply:
done